base_layer/wallet/tests/utxo_scanner/mod.rs

```diff
@@ -227,6 +232,9 @@ pub struct TestBlockData {
 /// Generates a set of block headers and key manager outputs for each header. The `birthday_offset` specifies at which
 /// block in the `num_block` the birthday timestamp will have passed i.e. it occured during the previous block period.
 /// e.g. with `num_blocks` = 10 and `birthday_offset` = 5 the birthday timestamp will occur between block 4 and 5
+
+// casting wrap around here is okay
```

Comment:
Feels vague. Maybe a little insight into _why_ it's okay here for people who may not know.

Reply:
added more detail to the comment